pkg/api/v1/BUILD

```diff
@@ -0,0 +1,38 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library")
+
+go_library(
+    name = "go_default_library",
+    srcs = ["zz_generated.conversion.go"],
+    importpath = "k8s.io/kubernetes/pkg/api/v1",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//pkg/api:go_default_library",
```

Comment:
this is the symptom of the build failure: there's no `go_default_library` in `pkg/api`, because there are no go files there

Reply:
Shouldn't this be fixed by running `hack/update-bazel.sh`?

Reply:
`hack/update-bazel.sh` uses whatever is specified in the go files. The go files import `k8s.io/kubernetes/pkg/api`, so the `BUILD` file is currently "correct".

Reply:
@ixdy so how do i fix this ? I have run update-bazel.sh and it doesnt do anything . i dont see any code that imports pkg/api  and i have already rebased , so what else do i need to do. I will update the developer guide if someone can point me in the right direction

Reply:
There is only one file in that library, so it must be that file: `zz_generated.conversion.go`, and that's exactly where @ixdy said "and here's the problem line".